app/components/molecules/navbar/navbar.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import "./navbar.scss";
+import NavItem from '../../atoms/navitem/navitem';
+
+interface NavItem {
+  label: string;
+  href: string;
+}
+
+interface NavbarProps {
+  navItems: NavItem[];
+  currentActivePage: string;
+}
+
+const Navbar: React.FC<NavbarProps> = ({ navItems, currentActivePage }) => {
+
+  return (
+    <nav className='navbar'>
+      <ul>
+        {navItems.map((item) => (
+          // Check if the href of the item matches the currentActivePage
+          item.href === currentActivePage && (
```

Comment:
Considering we are using Next.js, how come we aren't using the Next.js native navigation: https://nextjs.org/docs/app/building-your-application/routing/linking-and-navigating

Reply:
I think I fixed this in the dashboard branch.